performance_test/CMakeLists.txt

```diff
@@ -155,6 +156,17 @@ set(sources
     src/utilities/statistics_tracker.hpp
 )
 
+#ROS2 Waitset
+option(PERFORMANCE_TEST_POLLING_SUBSCRIPTION_ENABLED "Enable Polling Subscription Plugin to be available only
```

Comment:
and divide this line so it's not longer than 100chars.

Reply:
Corrected this.